examples/evp_1d_waves_on_a_string/waves_on_a_string.py

```diff
@@ -67,3 +67,11 @@
 plt.tight_layout()
 plt.savefig('eigenvalue_error.pdf')
 
+plt.figure(figsize=(6, 4))
+for n, idx in enumerate(np.argsort(solver.eigenvalues)[:5], start=1):
+    solver.set_state(idx, solver.subsystems[0])
+    plt.plot(x, u['g'].real.data, label=f"n={n}")
```

Comment:
Might be good to do `(u['g']/np.max(u['g'])).real` to normalize the amplitude and phase of the eigenvectors. It doesn't make much difference here, but is good practice.